packages/feedback-service/src/schema.ts

```diff
@@ -5,18 +5,10 @@ const FeedbackSchema: Schema = new Schema({
   experience: String,
   ticketID: String,
   spa: String,
-  timestamp: {
-    createdAt: { type: Date, default: Date.now },
-    createdBy: {
-      kerberosID: String,
-      name: String,
-    },
-    modifiedAt: Date,
-    modifiedBy: {
-      kerberosID: String,
-      name: String,
-    }
-  },
+  createdAt: { type: Date, default: Date.now },
+  createdBy: { type: String, ref: 'User' },
+  modifiedAt: Date,
```

Comment:
And change this to `updatedOn`
```suggestion
  updatedOn: { type: Date, default: Date.now },
```